ostelco-ios-client/Network/AppErrors.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  AppErrors.swift
+//  ostelco-ios-client
+//
+//  Created by Prasanth Ullattil on 30/04/2019.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import Foundation
+import Crashlytics
+
+class ApplicationErrors {
```

Comment:
@designatednerd @havard024 Is this a good way to report non fatal errors to crashalytics?

Reply:
Yeah I think so, adding a wrapper like this will be helpful so if we ever need to change the underlying error reporting method it's just in one place.

Reply:
Agreed, this seems fine, verified that they are visible in crashlytics for firebase